applications/users/controllers.py

```diff
@@ -103,6 +104,10 @@ def is_user_authorized_to_update_group(request, username, group_id):
 def get_user(request, email):
 	return db.get_user(request.db_session, email) if email is not None else None
 
+def get_user_profile(request, email):
```

Comment:
`get_user` returns only one value but `get_user_profile` returns two values, i.e. the user object and the auth_token. So their uses will be different from each other. I think keeping this way will be better as we won't have to change everywhere the way `get_user` will be called (accepting two return values). If there could be a better approach, please mention.